src/content/doc-sdk-javascript/index.mdx

```diff
@@ -97,6 +97,15 @@ To contribute to the SDK code, submit an Issue or Pull Request in the [surrealdb
             dark: DarkReact,
         }}
     />
+    <IconBox
```

Comment:
todo: also need to update this page `docs/sdk/javascript/frameworks/`

Reply:
Added!